src/main/java/it/gov/pagopa/reward/controller/InstrumentApiController.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.reward.controller;
+
+import it.gov.pagopa.reward.dto.HpanInitiativeBulkDTO;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import reactor.core.publisher.Mono;
+
+@RequestMapping("/reward")
```

Comment:
move here the more restrictive, yet general path, that could be the common prefix to each eventually added API
```suggestion
@RequestMapping("/reward/instrument")
```